modules/users/manifests/russellgarner.pp

```diff
@@ -1,6 +1,8 @@
 # Creates the russellgarner user
 class users::russellgarner {
+  #Once PR has been merged delete this file
   govuk_user { 'russellgarner':
+    ensure   => absent,
```

Comment:
This `ensure => absent` won't be run because he's been removed from the lists above.